packages/esm-service-queues-app/src/config-schema.ts

```diff
@@ -154,20 +160,24 @@ export const configSchema = {
   },
   contactAttributeType: {
```

Comment:
Tangential to your changes, but it looks like the `contactAttributeType` property in the `ConfigObject` interface we export is wrongly typed as `Array<string>` instead of `string`.